src/main/java/com/tripply/booking/entity/Flight.java

```diff
@@ -0,0 +1,52 @@
+package com.tripply.booking.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+
+@Data
+@Entity
+@Table(name = "flights")
+public class Flight {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long flightId;
+
+    @Column(nullable = false, unique = true)
```

Comment:
what is airline and why its unique ?

Reply:
I have added a new variable called flightNo to represent the flight number and made it unique. and airline represents the name of the flight.